src/apof/portal/static/portal/css/topbar.css

```diff
@@ -61,6 +73,19 @@
 }
 
 .navbar-brand a{
-    color: white;
     text-decoration: none;
+
```

Comment:
It increases readability. Those lines are placed to separate sertain parts of the CSS declaration.  Related property declarations should be grouped together following the order:
    Positioning
    Box model
    Typographic
    Visual

for example:
`

.declaration-order {
  /* Positioning */
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 100;

  /* Box-model */
  display: block;
  float: right;
  width: 100px;
  height: 100px;

  /* Typography */
  font: normal 13px "Helvetica Neue", sans-serif;
  line-height: 1.5;
  color: #333;
  text-align: center;

  /* Visual */
  background-color: #f5f5f5;
  border: 1px solid #e5e5e5;
  border-radius: 3px;

  /* Misc */
  opacity: 1;
}

`